mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -97,7 +97,6 @@ final class GBTClassifier @Since("1.4.0") (
 
   @Since("1.4.0")
   override def setSeed(value: Long): this.type = {
```

Comment:
This whole method can be removed now.  It was only overridden to log a warning before.